re:Invent 2024 Blog Post (#3388)
FusionAuth/fusionauth-site: astro/src/content/blog/fusionauth-reinvent-2024.mdx

---
publish_date: 2024-11-15 

> title: "FusionAuth at AWS re:Invent 2024"

Reviewer: This uppercases the `r` in `re:Invent`. Don't know any way around it because it is a CSS issue. So I guess it is okay.

description: Join us at AWS re:Invent 2024. Meet the team, get demos, and win some great prizes.
authors: Dan Stotts
image: /img/blogs/reinvent-2024/re-invent-2024.png
categories: Events

> tags: reinvent, 2024 events, fusionauth

Reviewer: ```suggestion
tags: reinvent, 2024 events, fusionauth, aws, las vegas
```

excerpt_separator: "{/* more */}"
---

> We’re thrilled to announce that FusionAuth will be at [AWS re:Invent 2024](https://reinvent.awsevents.com/), and we can’t wait to connect with you! We’ve planned an array of activities and opportunities for engagement throughout the event.

Reviewer: In the past we've also highlighted some interesting sessions. See https://fusionauth.io/blog/fusionauth-re-invent-2023

Don't know if we want to do that here. Happy to provide those if you want.

Author: If you have them, please do send some my way. I'm just posting what Dan Stotts sent to me, but I'm happy to add them.

Reviewer: Here are four I'm interested in attending. You could say something like: "we're of course interested in some of the sessions too. Here are four that some team members found especially interesting"

SEC346: Customer insights on application authorization 
SAS312: Next-generation SaaS tenant isolation strategies 
DAT335: Safer, simpler, faster upgrades with Amazon RDS Blue/Green Deployments 
KUB323: Observability strategies for Amazon EKS workloads

## Ways to Engage:

> **1. Find Us at Our Booth**

Reviewer: Any reason not to make these headings?

```suggestion
### 1. Find Us at Our Booth
```

> Swing by [Booth #1079](https://awsreinvent24.mapyourshow.com/8_0/exhview/index.cfm?orsearchtype0=exhibitor&orsearchvalue0=1112&orsearchdisplay0=Exhibitor&orsearchvaluedisplay0=FusionAuth) during Expo hours to meet our team, see a demo, and learn how FusionAuth can supercharge your authentication and authorization needs.

Reviewer: ```suggestion
Swing by [Booth #1079](https://awsreinvent24.mapyourshow.com/8_0/exhview/index.cfm?orsearchtype0=exhibitor&orsearchvalue0=1112&orsearchdisplay0=Exhibitor&orsearchvaluedisplay0=FusionAuth) during Expo hours to meet our team, see a demo, and learn how FusionAuth can help with your authentication and authorization needs. You'll have to zoom in a bit, we're below the "Community and Impact Pavillion" on that map.
```

supercharge feels over the top, imo.

**Booth Hours (PST):**

- **Monday, December 2**: 4 PM - 7 PM (Welcome Reception)
- **Tues & Weds, Dec. 3&4**: 10 AM - 6 PM
- **Thursday, Dec. 5**: 10 AM - 5 PM

**Daily Prizes: Enter & Win!**

Stop by our booth and chat with a FusionAuth representative to enter our daily drawing for a chance to win amazing prizes like:

- Nintendo Switch
- Keychron Keyboard
- LEGO Sets

Winners will be drawn daily. Prizes awarded Monday through Thursday. You must be present to win!

**2. Book Time with an Expert**

> Got questions about authentication and authorization? Need some best practices or just want to swap stories? Our experts are here for you. Plus, they might even throw in a few Chuck Norris jokes.

Reviewer: Are we leaning into this? Is this a free support session? What is this going to offer beyond a demo? Do you want to drop the fact that it will be with the CTO or senior member of the engineering team (if that is true)? How can you make this something someone will actually want to do?

How long is it?

Author: I shot this over to Dan S for clarification.

> [Schedule a Session](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ0fyoPWqEcV5AG26KmgOLqRgN92zefevxvEkkFy2hwcO6IEkERGkzvq-CULX1o53Hcx_Z3nA1Qb)

Reviewer: ```suggestion
[Schedule a Session With an Authentication Expert](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ0fyoPWqEcV5AG26KmgOLqRgN92zefevxvEkkFy2hwcO6IEkERGkzvq-CULX1o53Hcx_Z3nA1Qb)
```

**3. Join FusionAuth at The Sphere**

**Tuesday, December 3 | 7 PM - 10 PM**

Take a visual journey across our planet at The Sphere’s “Postcard From Earth,” directed by Darren Aronofsky. This immersive show combines stunning visuals and captivating storytelling. Join us for an unforgettable evening of entertainment, food, and drinks.

> [Signup Now](https://share.hsforms.com/1RmUHh6EsT3mk5DiLpGnIKQdlrn?__hstc=186035894.ac043d7cfa24c3cdee097cdebf8c80da.1731432507133.1731432507133.1731512687824.2&__hssc=186035894.1.1731512687824&__hsfp=301312424)

Reviewer: Do you need the __hstc stuff parameters? I don't know if those are throwing things off or required, but the form renders fine for me if the URL is `https://share.hsforms.com/1RmUHh6EsT3mk5DiLpGnIKQdlrn`.

**4. Exclusive VIP Round of Golf with Our CEO**

**Tuesday, December 3 | 9:15 AM**

Join FusionAuth CEO Brian Pontarelli for 18 holes at the Wynn Golf Club, a Tom Fazio-designed championship course. Lush greens, rushing streams, and some friendly competition await!

> [Request Your Invite](https://share.hsforms.com/1RmUHh6EsT3mk5DiLpGnIKQdlrn?__hstc=186035894.ac043d7cfa24c3cdee097cdebf8c80da.1731432507133.1731432507133.1731512687824.2&__hssc=186035894.1.1731512687824&__hsfp=301312424)

Reviewer: same q about parameters.
